deploy/kustomize/overlays/cloudbeat-vanilla-agent/README.md

```diff
@@ -0,0 +1,65 @@
+# Kustomize Vanilla Agent
+
+This manifests are used to deploy the agent on local kind cluster with dev image of the agent.
+So that later we can inject into that agent a custom binray of cloudbeat and run E2E flow.
+
+## How to use it
+
+### Build the Agent
+Note: this step might take a while
+**From agent repo**
+```bash
+DEV=true PACKAGES="docker" EXTERNAL=true SNAPSHOT=true mage -v package
```

Comment:
You have a make target inside cloudbeat to build both agent and cloudbeat from your source code
https://github.com/DaveSys911/cloudbeat/blob/b37a28ef84613bb57a9e371e154c148e15880d2b/Makefile#L76